backend/app/api/endpoints/adapter/chat.py

```
@@ -303,12 +379,14 @@ async def _create_task_and_subtasks(
         next_message_id = existing_subtasks[0].message_id + 1
         parent_id = existing_subtasks[0].message_id
 
-    # Create USER subtask
+    # Create USER subtask (always created)
+    from app.models.subtask import SenderType
```

Ruff (0.14.8)

Redefinition of unused `SenderType` from line 22

Remove definition: `SenderType`

(F811)